README.md

```diff
@@ -1,126 +1,110 @@
-| [![Screenshot of store homepage](./docs/img/online-boutique-frontend-1.png)](./docs/img/online-boutique-frontend-1.png) | [![Screenshot of checkout screen](./docs/img/online-boutique-frontend-2.png)](./docs/img/online-boutique-frontend-2.png) |
+| [![Screenshot of store homepage](/docs/img/online-boutique-frontend-1.png)](/docs/img/online-boutique-frontend-1.png) | [![Screenshot of checkout screen](/docs/img/online-boutique-frontend-2.png)](/docs/img/online-boutique-frontend-2.png) |
 
-## Quickstart (GKE)
+## Interactive quickstart (GKE)
 
 [![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fmicroservices-demo&shellonly=true&cloudshell_image=gcr.io/ds-artifacts-cloudshell/deploystack_custom_image)
 
-1. **[Create a Google Cloud Platform project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project)** or use an existing project. Set the `PROJECT_ID` environment variable and ensure the Google Kubernetes Engine and Cloud Operations APIs are enabled.
-
-```
-PROJECT_ID="<your-project-id>"
-gcloud services enable container.googleapis.com --project ${PROJECT_ID}
-```
-
-2. **Clone this repository.**
+## Quickstart (GKE)
 
-```
-git clone https://github.com/GoogleCloudPlatform/microservices-demo.git
-cd microservices-demo
-```
+1. Ensure you have the following requirements:
+   - [Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project).
+   - Shell environment with `gcloud`, `git`, and `kubectl`.
 
-3. **Create a GKE cluster.**
+2. Clone the repository.
 
-- GKE autopilot mode (see [Autopilot
-overview](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-overview)
-to learn more):
+   ```sh
+   git clone https://github.com/GoogleCloudPlatform/microservices-demo
+   cd microservices-demo/
+   ```
 
-```
-REGION=us-central1
-gcloud container clusters create-auto onlineboutique \
-    --project=${PROJECT_ID} --region=${REGION}
-```
+3. Set the Google Cloud project and region and ensure the Google Kubernetes Engine API is enabled.
 
-- GKE Standard mode:
+   ```sh
+   gcloud config set project <PROJECT_ID>
+   gcloud config set compute/region us-central1
+   gcloud services enable container.googleapis.com
+   ```
 
-```
-ZONE=us-central1-b
-gcloud container clusters create onlineboutique \
-    --project=${PROJECT_ID} --zone=${ZONE} \
-    --machine-type=e2-standard-2 --num-nodes=4
-```
+   Substitute `<PROJECT_ID>` with the ID of your Google Cloud project.
 
-4. **Deploy the sample app to the cluster.**
+4. Create a GKE cluster and get the credentials for it.
 
-```
-kubectl apply -f ./release/kubernetes-manifests.yaml
-```
+   ```sh
+   gcloud container clusters create-auto online-boutique
```

Comment:
**Issue:** This `gcloud container clusters create-auto online-boutique` command gave me an error:
```
ERROR: (gcloud.container.clusters.create-auto) ResponseError: code=400, message=Autopilot clusters must be regional clusters.
```
This might be specific to my `gcloud` configuration — but I can see our users encountering the same issue.
Let's specify the region:
```
gcloud container clusters create-auto online-boutique --region=us-central1
```

Reply:
Fixed!